packages/observability-node/src/index.ts

```diff
@@ -1 +1,2 @@
-export * from "./grafana";
+export * from "./logger";
+export * from "./graphql/getUserIdOrThrow";
```

Comment:
Do we want to export this? It would be exported in shared package eventually but not here right?

Reply:
My understanding was that I should add whatever I need to write or use the observability package in the directory (`packages/observability-node/*`).

https://linear.app/replay/issue/PRO-629/replay-cli-project-should-better-support-shared-inlined-code

Reply:
Oh nevermind, I thought `getUserIdOrThrow` isn't being used outside of `observability-node`. I was wrong.

Reply:
RE: PRO-629
> Share code by copy-pasting it for now, leaving a TODO comment that refers to this issue so that we can consolidate after this change has been made:
>
> ```js
> // TODO [PRO-629] Move this into the "shared" package
> ```

FWIW my intent with that comment was to "share" by copy-pasting between packages for now, so that we didn't end up publishing anything that we'd later want to remove (and require a breaking chagne)

Reply:
I think the `observability-node` package could actually just become the shared package. I would want to rename it, of course. What do you think about that?

Reply:
I was leaning that way too but @Andarist (I think?) mentioned that we may want an observability/logging type package to be shared between things that use it so it could coordinate session data. (I am not representing his argument well sry)

I don't know how likely that is to happen unless we are very strict about our dependency version ranges. And I'm not sure if it really buys us that much that I can think of so maybe I still lean towards shared but it seems like something we may want to chat about (the three of us)